src/components/stopDrive.ts

```diff
@@ -0,0 +1,21 @@
+import GarageAPI from './GarageAPI';
+
+async function stopDrive(
+  requestAnimationIds: {[key: string]: number},
+  carId: number,
+  carImage: HTMLElement,
+) {
+  const garageAPI = new GarageAPI();
+  const raceButton = document.querySelector('#race');
+  const raceResetButton = document.querySelector('#reset');
+
+  cancelAnimationFrame(requestAnimationIds[carId]);
+  const response = await garageAPI.startStop(+carId, 'stopped');
+  // eslint-disable-next-line no-param-reassign
+  carImage.style.transform = `translateX(${response.velocity}px)`;
+  document.getElementById(`stop.${carId}`)?.setAttribute('disabled', 'true');
```

Comment:
creation of `stop.${carId}`/`start.${carId}` should be extracted into a helper function